terraform/010-cluster/vars.tf

```diff
@@ -115,3 +115,9 @@ variable "log_retention_in_days" {
   type        = number
   default     = 30
 }
+
+variable "enable_ec2_detailed_monitoring" {
+  description = "Whether to enable detailed monitoring for EC2 instances"
+  type        = bool
+  default     = false
```

Comment:
Actually, I think this should probably be `true` since this is not a root module. Changing it to `false` from the previous default would be a breaking change.

Reply:
yeah, updated it now.